README.md

```diff
@@ -89,7 +89,7 @@ non-WGSL shaders if you're running on WebGPU.
 
 WGSL is always supported by default, but GLSL and SPIR-V need features enabled to compile in support.
 
-To enable SPIR-V shaders, enable the `spirv` feature of wgpu.  
```

Comment:
What happened here? These need to be preserved as they need to be on separate lines.

Reply:
Looks like editorconfig messed it up.